docs/data-sources/tasks.md

```diff
@@ -2,42 +2,967 @@
 page_title: "snowflake_tasks Data Source - terraform-provider-snowflake"
 subcategory: ""
 description: |-
-  
+  Datasource used to get details of filtered tasks. Filtering is aligned with the current possibilities for SHOW TASKS https://docs.snowflake.com/en/sql-reference/sql/show-tasks query. The results of SHOW and SHOW PARAMETERS IN are encapsulated in one output collection tasks.
 ---
 
-# snowflake_tasks (Data Source)
+!> **V1 release candidate** This data source was reworked and is a release candidate for the V1. We do not expect significant changes in it before the V1. We will welcome any feedback and adjust the data source if needed. Any errors reported will be resolved with a higher priority. We encourage checking this data source out before the V1 release. Please follow the [migration guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0920--v0930) to use it.
 
+# snowflake_tasks (Data Source)
 
+Datasource used to get details of filtered tasks. Filtering is aligned with the current possibilities for [SHOW TASKS](https://docs.snowflake.com/en/sql-reference/sql/show-tasks) query. The results of SHOW and SHOW PARAMETERS IN are encapsulated in one output collection `tasks`.
```

Comment:
nit (general): we should think of being consistent: data source or datasource (above in v1 release candidate note there is data source and in this paragraph datasource). Let's maybe add it to our task to correct the docs before the v1?

Reply:
https://developer.hashicorp.com/terraform/language/data-sources here it's separate, so maybe let's do the same? Changed.